src/screens/login/login.js

```diff
@@ -0,0 +1,32 @@
+import './login.css';
+import loginContent from './login.hbs';
+
+export default class LoginScreen {
+  constructor(parent, email, password) {
+    this.parent = parent;
+    this.email = email;
+    this.password = password;
+  }
+
+  render(parent, visibility) {
```

Comment:
Instead of passing string as a second parameter I would suggest passing an object called 'props' or 'options' with any properties that the component needs.